sdk/keyvault/Azure.Security.KeyVault.Certificates/src/ImportCertificateOptions.cs

```diff
@@ -79,6 +80,12 @@ public ImportCertificateOptions(string name, byte[] certificate)
         /// </summary>
         public IDictionary<string, string> Tags => LazyInitializer.EnsureInitialized(ref _tags);
 
+        /// <summary>
+        /// Gets or sets a value indicating whether the certificate chain preserves its original order.
+        /// The default value is false, which sets the leaf certificate at index 0.
+        /// </summary>
+        public bool? PreserveCertOrder { get; set; }
```

Comment:
```suggestion
        public bool? PreserveCertificateOrder { get; set; }
```

Please see previous comments for context.